src/index.css

```
@@ -73,7 +76,7 @@ h4 {
 
 h3 {
   font-size: 1rem;
-  line-height: 1,5rem;
+  line-height: 1, 5rem;
```

just delete the extra space
```suggestion
  line-height: 1,5rem;
```